core/src/main/java/org/wso2/msf4j/MicroservicesRunner.java

```diff
@@ -156,46 +173,55 @@ protected void configureTransport(int... ports) {
+        serverConnectors.addAll(httpServerConnectorProvider.initializeConnectors(transportsConfiguration));
+        serverConnectors.forEach(serverConnector -> serverConnector.setMessageProcessor(new MSF4JMessageProcessor()));
     }
 
     /**
      * Method to configure transports.
      */
     protected void configureTransport() {
-        TransportsConfiguration trpConfig = YAMLTransportConfigurationBuilder.build();
-        Set<ListenerConfiguration> listenerConfigurations = trpConfig.getListenerConfigurations();
-        HTTPTransportContextHolder httpTransportContextHolder = HTTPTransportContextHolder.getInstance();
-        httpTransportContextHolder.setHandlerExecutor(new HandlerExecutor());
-
-        Set<TransportProperty> transportProperties = new HashSet<>();
-        TransportProperty transportProperty = new TransportProperty();
-        int bossGroupSize = Runtime.getRuntime().availableProcessors();
-        transportProperty.setName(Constants.SERVER_BOOTSTRAP_BOSS_GROUP_SIZE);
-        transportProperty.setValue(bossGroupSize);
-        TransportProperty workerGroup = new TransportProperty();
-        int workerGroupSize = Runtime.getRuntime().availableProcessors() * 2;
-        workerGroup.setName(Constants.SERVER_BOOTSTRAP_WORKER_GROUP_SIZE);
-        workerGroup.setValue(workerGroupSize);
-        transportProperties.add(transportProperty);
-        transportProperties.add(workerGroup);
-        httpTransportContextHolder.setMessageProcessor(new MSF4JMessageProcessor());
-        for (ListenerConfiguration listenerConfiguration : listenerConfigurations) {
-            HTTPTransportListener listener =
-                    new HTTPTransportListener(transportProperties, Collections.singleton(listenerConfiguration));
-            transportManager.registerTransport(listener);
-            DataHolder.getInstance().getMicroservicesRegistries().put(listenerConfiguration.getId(), msRegistry);
-        }
+        //TransportsConfiguration transportsConfiguration = YAMLTransportConfigurationBuilder.build();
+        /*ServerConnectorController serverConnectorController = new ServerConnectorController(transportsConfiguration);
+        serverConnectorController.start();
+        transportsConfiguration.getListenerConfigurations()
+                               .forEach(listenerConfiguration -> {
+                                   listenerConfiguration.setBindOnStartup(false);
+                                   DataHolder.getInstance().getMicroservicesRegistries()
+                                             .put(listenerConfiguration.getId(), msRegistry);
+                               });
+        HTTPServerConnectorProvider httpServerConnectorProvider = new HTTPServerConnectorProvider();
+        serverConnectors.addAll(httpServerConnectorProvider.initializeConnectors(transportsConfiguration));
+        serverConnectors.forEach(serverConnector -> serverConnector.setMessageProcessor(new MSF4JMessageProcessor()));*/
```

Comment:
Are these comments required? Ideally commented code should be removed. Having commented code is considered as a code smell